matchMinima.py

```diff
@@ -432,13 +395,15 @@ def getRatioTimes(allMolTimes, zeroes):
 
     """
 
+#M Unrelated: You're compring times spent in each minima? What for?
     relByFile = []
     sdByFile = []
     for i, molist in enumerate(allMolTimes):
         molTimes = []
         molStds = []
         for j, filelist in enumerate(molist):
             rels = np.asarray(filelist)/np.asarray(molist[0])
+            #M I trust this is safe?
```

Comment:
Does eliminating nan's like that bias your results? I assume some conformers don't finish optimization because of a timeout.